doc/makeflow.html

```diff
@@ -410,14 +410,36 @@ <h2>Table of Contents</h2>
 
 <p>Use the <tt>-T pbs</tt> option to submit jobs to the <a href=http://www.arc.ox.ac.uk/content/pbs>Portable Batch System</a> or compatible systems.</p>
 
+<p>This will add the values for cores, memory, and disk. These values will be added onto <tt>qsub</tt> in this format:<br>
+<tt>-l nodes=1:ppn=$CORES,mem=$MEMORY,file=$DISK</tt></p>
```

Comment:
should it be 

mem=${MEMORY}mb  (same for file).

We also want to be explicit in that measurements are in MB.